run/Flan/Modern Weapons Pack/vehicles/Abrams.txt

```diff
@@ -32,12 +32,30 @@ FuelTankSize 2000
 //Inventory Slots
 CargoSlots 0
 ShellSlots 5
+
```

Comment:
Those blank lines were there for very obvious separation to show what had been changed. They shouldn't be present in this version.